src/SbManager/Scripts/manager/homeController.js

```diff
@@ -1,11 +1,17 @@
-﻿$app.controller('homeController', ['$scope', function ($scope) {
+﻿$app.controller('homeController', ['$scope', '$route', '_', function ($scope, $route, _) {
+
+    $scope.deadletterFilterEnabled = $route.current.$$route.deadletterFilter;
+
     $scope.refresh = function () {
         $scope.model = null;
+
         $.ajax({
             url: window.applicationBasePath + "/api/v1/busmanager/",
             dataType: 'json',
             success: function(d) {
                 $scope.model = d;
+                $scope.queuesWithDeadlettersCount = d.Queues.reduce((c, q) => c + (q.DeadLetterCount ? 1 : 0), 0);
```

Comment:
This will break some browsers but we haven't solved #65 yet so I'm ok with it for now.